src/modules/Dynamic/Module.js

```diff
@@ -25,48 +25,66 @@ module.exports = function Dynamic(options,UI) {
       eval(func);
       return f;
     }
-
+    
     var channels = ['red', 'green', 'blue', 'alpha'];
-
+    
     channels.forEach(function(channel) {
       if (options.hasOwnProperty(channel)) options[channel + '_function'] = generator(options[channel]);
       else if (channel === 'alpha')        options['alpha_function'] = function() { return 255; }
       else                                 options[channel + '_function'] = generator(options.monochrome);
     });
-
+    
     function changePixel(r, g, b, a) {
+
+      /* neighbourpixels can be calculated by
+      
+       this.getNeighbourPixel.fun(x,y) 
```

Comment:
Sorry, just one more! could we go with the American English spelling of `Neighbor` here? OR alias it so we can use both? 👍 

And just curious, sorry if it's a silly question but why does `getNeighborPixel` have to be an object -- can't it just be `getNeighborPixel()`?

Reply:
@jywarren actually i have made it an object because pixelmanipulation remaps the options.getNeighbourpixel.fun to a new function which internally calls the original getneighbourpixels with the value of `pixels` it gets by closure. This remapping won't be possible if it's a function since we can't just say
```
 options.getneighbourspixel = something else
```
And it feels kind of natural to use the same function defined in the same file
To avoid this we have 2 options either we move all the logic inside pixelmanipulation or use a different name than getneighbourpixels for actual function that will be called

Reply:
@jywarren i just opened another pull request doing it have getNeighbourPixel as function, #207 you can merge that if that looks more natural to you, thanks😁

Reply:
OK I left a comment on #207 -- that does look good!